AdventOfCode/Config.cs

```diff
@@ -58,9 +58,11 @@ public int[] Days
 
         void setDefaults()
         {
-            if(Cookie == default(string)) Cookie = "";
-            if(Year == default(int)) Year = DateTime.Now.Year;
-            if(Days == default(int[])) Days = (DateTime.Now.Month == 12) ? new int[] { DateTime.Now.Day } : new int[] { 0 };
+            //Make sure we're looking at EST, or it might break for most of the US
```

Comment:
I'm unsure if I understand the breakage here. Do you mean that since EST will reach a given date before the rest of the US, most of the country will be lagging behind unless they change the day field manually (I assumed EST was the eastern-most timezone in the US)? I guess it sorta begs the question on what role this default day fetching should fill.

As a European myself I've always been 6-7 hours ahead and the puzzles were always available when I got up in the morning, so I haven't actually given much thought to the cases where a few hours back or forth matter (aside from assuming that anyone doing the time trial would set the days manually).

Reply:
The first one. At midnight on the US East Coast (EST) it is still "the day before" for 75% of the mainland. In Seattle for example it's only 21:00 because it's in PST. Makes scheduling meetings across the country interesting sometimes. Washington DC is going to lunch at the same time that California is just clocking in for the day.

Reply:
Heh, yeah, timezones are fun. Abiding by EST globally is what makes most sense I suppose.

Reply:
cbb2d2e